### DISCUSSION TITLE
Most performant usage of arrayjoin - few files with high res vs. many files with low res?

### DISCUSSION THREAD

**Discussion by liveman1** [libvips/libvips#2617] [Q&A]
Hi,

What is the best way using arrayjoin to create DZ images?
Is it more performant to join a small amount of images with a high resolution or to join a large amount of images with a small resolution?

thanks for sharing your experience!

**Comment by jcupitt** [comment]
Hi @liveman1,

Fewer larger tiles would be faster, but need more memory. I doubt if it'll make a huge difference though.
